### PR TITLE
Bring pcornet v3 field attributes into compliance

### LINKED ISSUE #136: Updates to PCORnet v3 field lengths
PCORnet is getting stricter with their field lengths and we need to use their proscribed values instead of the guesstimates we have been using (which we only used because they didn't publish length values until recently). The relevant links are:

Full PCORnet CDM specification (not updated with errata): http://www.pcornet.org/pcornet-common-data-model/

PCORnet CDM errata: https://github.com/CDMFORUM/CDM-ERRATA

### PR DESCRIPTION
**Pull request by aaron0browne**
Mostly varchar and numeric fields. Note that uniformly, if fields are
specified in the pcornet doc as "NUMBER(8)", they are here given
"number,,10,2", to allow for some decimal specification and avoid the
application of a default scale which would occur if the scale were
left off or specified as 0.

Fixes #136

Signed-off-by: Aaron Browne <aaron0browne@gmail.com>

### CODE REVIEW

**Comment by bruth** [comment]
Looks good.

**Comment by aaron0browne** [comment]
For some reason it failed to produce valid DDL through the service; I'm looking into it.

File: pcornet/v3/schema/harvest.csv [hunk 1]
```diff
@@ -4,27 +4,27 @@ pcornet,v3,harvest,network_name,string,20,,,
...
-pcornet,v3,harvest,lab_order_date_mgmt,string,,,,
-pcornet,v3,harvest,specimen_date_mgmt,string,,,,
-pcornet,v3,harvest,result_date_mgmt,string,,,,
-pcornet,v3,harvest,measure_date_mgmt,string,,,,
-pcornet,v3,harvest,onset_date_mgmt,string,,,,
-pcornet,v3,harvest,report_date_mgmt,string,,,,
-pcornet,v3,harvest,resolve_date_mgmt,string,,,,
-pcornet,v3,harvest,pro_date_mgmt,string,,,,
+pcornet,v3,harvest,birth_date_mgmt,string,2,,,
+pcornet,v3,harvest,enr_start_date_mgmt,string,2,,
```
Comment (aaron0browne) [inline pcornet/v3/schema/harvest.csv]: Here's the error. I unintentionally removed some commas from this line and those following. Fixing now.

**Comment by aaron0browne** [comment]
There we go, all hunky dory and ready for merge.

**Pull request merged**